contracts/AnchoredView/SymbolConfiguration.sol

```diff
@@ -14,15 +14,18 @@ contract SymbolConfiguration {
     address public constant cUsdtAnchorKey = address(1);
     address public constant cDaiAnchorKey = address(2);
 
+    /// @notice standard amount for the Dollar
+    uint constant oneDollar = 1e6;
+
     /// Address of the oracle key (underlying) for cTokens non special keyed tokens
     address public immutable cRepAnchorKey;
     address public immutable cWbtcAnchorKey;
     address public immutable cBatAnchorKey;
     address public immutable cZrxAnchorKey;
 
-    /// Frozen rices for SAI and eth, so no oracle key
-    uint public saiAnchorPrice = 5285551943761727;
-    uint public ethAnchorPrice = 1e18;
+    /// Frozen prices for SAI and eth, so no oracle key
```

Comment:
FYI I think /// is only for natspec comments

Reply:
I was wondering why it was 3...